wpa_supplicant: new article

mcpcpc opened the pull request.

dylanaraps commented on post-installation/wpa_supplicant.txt:
```diff
@@ -0,0 +1,92 @@
+At this point, you will want to create a wpa_supplicant file to store your 
+wireless network information and credentials:
+
+$ mkdir /etc/wpa_supplicant
```
`mkdir -p`.

dylanaraps commented:
This is really nice. Thanks

dylanaraps commented:
I think this makes much more sense to live in a category called `wifi`. I also want to write an `eiwd` page alongside this one. :+1:

mcpcpc commented:
> I think this makes much more sense to live in a category called `wifi`. I also want to write an `eiwd` page alongside this one. 👍

make sense. moved post-installation->wifi and updated the index.txt file.

dylanaraps commented on post-installation/wpa_supplicant.conf:
```diff
@@ -0,0 +1,10 @@
+# Allow users in the 'wheel' group to control wpa_supplicant  
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=wheel
+
+# Make this file writable for wpa_gui / wpa_cli
+update_config=1
+
+network={
+       ssid="<BSSID>"
+       psk="<PASSWORD>"
+}
```
This is probably better embedded in the page itself. Keeps things nice and self-contained.

dylanaraps commented on post-installation/wpa_supplicant.txt:
```diff
@@ -0,0 +1,92 @@
+Generating a Passphrase
+________________________________________________________________________________
+
+NOTE: This section is not required but HIGHLY recommend since storing your
```
`recommend` -> `recommended`

dylanaraps commented on post-installation/wpa_supplicant.txt:
```diff
@@ -0,0 +1,92 @@
+________________________________________________________________________________
+
+NOTE: This section is not required but HIGHLY recommend since storing your
+      password in clear text is not a good practice. 
```
`not a good practice` -> `not good practice`

dylanaraps commented on post-installation/wpa_supplicant.txt:
```diff
@@ -0,0 +1,92 @@
+can make use of wpa_passphrase, a command line tool which generates the minimal
+configuration needed by wpa_supplicant: 
+
+$ wpa_passphrase <BSSID> <PASSWORD>
```
`<BSSID>` (and all occurrences of `<word>`) will be interpreted as HTML by the browser and won't be displayed. You have two options here, using `&lt;`/`&gt;` or by swapping to a different character.  :+1:

dylanaraps commented on post-installation/wpa_supplicant.txt:
```diff
@@ -0,0 +1,92 @@
+Managed Wireless Connection via DHCPCD
+________________________________________________________________________________
+
+Assuming that dhcpd is already installed, busybox's runsv can be used to create 
```
`dhcpd` -> `dhcpcd`

dylanaraps commented:
Thanks. :+1:
